app/controllers/course/admin/admin_controller.rb

```diff
@@ -11,6 +11,15 @@ def update #:nodoc:
     end
   end
 
+  def destroy #:nodoc:
```

Comment:
@weiqingtoh sorry, sorry, was half asleep -- destroy success and destroy failure